cypress/integration/Homework/numbers.js

```diff
@@ -0,0 +1,124 @@
+const numberAnswerArray = [
```

Comment:
It looks like for the future it would be more readable to have a table with all test data. No need to change it now.